src/Article.js

```diff
@@ -2,15 +2,10 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 
 class Article extends Component {
-  constructor(props) {
-    super(props)
-
-    this.handleClick = this.handleClick.bind(this)
-  }
-
-  handleClick() {
+  handleClick = () => {
```

Comment:
If you want to avoid binding all the functions in your constructor - this is a cool trick that lets you autobind the function when you define it.

if you define the function as `popup = () => { }` instead of `popup() {}` the arrow function magically auto-binds the function to the class for you.